### PR TITLE
Upgrade to .net6

### PR DESCRIPTION
**Pull request by theace0296**
Went through all the requirements to switch to .net6, the main things that changed are the way you generate SHA1 hashes, and handle basic web requests. An addition is .net6 can natively handle/create symlinks.

### CODE REVIEW

File: FASTER/FASTER.csproj [hunk 1]
```diff
@@ -50,7 +52,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Autoupdater.NET.Official" Version="1.7.0" />
-    <PackageReference Include="BytexDigital.Steam" Version="0.8.0-alpha2" />
+    <PackageReference Include="BytexDigital.Steam" Version="0.8.1-development.1647207886" />
```
Comment (theace0296) [inline FASTER/FASTER.csproj:55]: Had to bump to a dev version of BytexDigital.Steam to support .net6.

**Comment by Foxlider** [comment]
1.8 BETA is already on .NET 6 with (most of) those changes though :(

**Comment by theace0296** [comment]
> 1.8 BETA is already on .NET 6 with (most of) those changes though :(

Ahh I see, I thought 1.8 was merged into master already, my bad!

**Comment by Foxlider** [comment]
Kinda was but only 1.8b was. I noticed I had a lot to fix before heading for 1.9. 

So I'm still working on that branch

**Pull request closed without merging**